README.md

```diff
@@ -1,3 +1,33 @@
+## Setup
+
+1. Ensure pre-commit hooks are setup for safe commits. See below section on "Setting up Git Guardian"
+2. Ensure node 14 is installed. Install and use nvm to manage multiple node versions.
+3. Run `npm i` to install required packages
+4. Ensure [Docker](https://www.docker.com/products/docker-desktop/) is installed
+5. Run `npm run dev:servies` to bring up the docker containers
+6. Run `npm run dev` to start the server
+
+## Setting Up Git Guardian
+
+1. Install GitGuardian
+
+```
+brew install gitguardian/tap/ggshield
+```
+
+2. Add the API Key to your shell startup env (e.g. in `.zshrc`)
+
+```
+export GITGUARDIAN_API_KEY=abc123
```

Comment:
I assume this API key is created by us manually on gitguardian portal, which is fine. But `abc123` is not exactly a good value for an API key, would it be easy to replace it with a more randomized value?